src/componentLibrary/CoinPair/CoinPair.tsx

```diff
@@ -1,5 +1,5 @@
 import { CurrencySymbols } from 'types/currency';
-import { CoinIcon } from '../';
+import { CoinIcon } from '..';
```

Comment:
I think we could use absolute import from `componentLibrary` here instead, but this was edited just by linter, right?

Reply:
Not sure what made this kind of change. I'm sure I did not make it manually myself.

Reply:
FYI: https://github.com/interlay/interbtc-ui/pull/367#discussion_r913568266

Reply:
@tomjeatt @crypto-engineer I just checked the code and different ways of importing lib components inside of library are used, so let's figure this out and update the imports to one consistent style :+1:

Reply:
@peterslany could you and Anton take a look at this? His editor is obviously doing something helpful that ours aren't :) don't mind what pattern we use, but would be good to have the tooling handle it for us.

Reply:
@crypto-engineer do you know what plugin/extension your IDE used to convert these imports? I think it would be useful to add it to global prettier config if it's possible :)

Reply:
I'm not sure. One thing I'm suspicious of is ESLint VSCode extension.